pkg/controller/operators/catalog/operator_test.go

```diff
@@ -1205,17 +1280,17 @@ func objectReference(name string) *corev1.ObjectReference {
 	return &corev1.ObjectReference{Name: name}
 }
 
-func yamlFromFilePath(fileName string) (string, error) {
+func yamlFromFilePath(t *testing.T, fileName string) string {
```

Comment:
Non-blocking nit: A few of these functions could probably be moved to an `e2e-util.go` file.

Reply:
Good idea! I think a follow-up would involve making a helper package that can be used across all of OLM's tests.